Configuration/TCA/Overrides/tx_telephonedirectory_domain_model_employee.php

```diff
@@ -1,14 +1,28 @@
 <?php
-if (!defined('TYPO3_MODE')) {
+
+/*
+ * This file is part of the package jweiland/telephonedirectory.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+if (!defined('TYPO3')) {
     die('Access denied.');
 }
 
-\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::makeCategorizable(
-    'telephonedirectory',
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+$GLOBALS['TCA']['tx_telephonedirectory_domain_model_employee']['columns']['additional_function'] = [
```

Comment:
With deprecated/removed makeCategorizable we should add category columns directly into the TCA instead of overwriting it here.